src/common/Connection.ts

```diff
@@ -167,8 +167,8 @@ export class Connection extends EventEmitter {
   }
 
   /**
-   * @param {Protocol.Target.TargetInfo} targetInfo
-   * @returns {!Promise<!CDPSession>}
+   * @param targetInfo - Gets targetInfo
```

Comment:
targetInfo here is a param the user provides, so "Gets targetInfo" doesn't quite fit. If you're not sure what to document for this param, just leave it as:

```
@param targetInfo
```